geometry/create_capacitance_file_theta.py

```diff
@@ -0,0 +1,381 @@
+
+# 1*15mm + 11*35mm = 400
+# activeTotal = 400.0
+# inclinedTotal = 564.964
+# tracesPerLayer = [6, 1, 1, 0, 0, 1, 2, 3, 4, 5, 6, 7] # only one trace for strip layer because 4 cells instead of one
+# ncells_strip_layer = 4.0
+# careful, this is not really the radial spacing, it is, after dilution, the spacing in the parallel direction --> radial depth spacing will not be constant
+# readoutLayerRadialLengths = [1.500000] * 1 + [3.500000] * 11
+# Detector
+# rmin = 2160
+# Nplanes = 1536
+# inclination_degree = 50
+# angle = inclination_degree / 180. * pi #inclination angle inn radian
+# passiveThickness = 2.0 #mm
+
+# latest
+activeTotal = 400.5
+inclinedTotal = 565.86
+tracesPerLayer = [6, 1, 1, 0, 0, 1, 2, 3, 4, 5, 6, 7] # only one trace for strip layer because 4 cells instead of one
+ncells_strip_layer = 4.0
+# careful, this is not really the radial spacing, it is, after dilution, the spacing in the parallel direction --> radial depth spacing will not be constant
+readoutLayerRadialLengths = [1.500000] * 1 + [3.500000] * 11
+# Detector
+rmin = 2172.8
+Nplanes = 1545
+inclination_degree = 50
+angle = inclination_degree / 180. * pi #inclination angle inn radian
+passiveThickness = 2.0 #mm
+#Segmentation
+deltaTheta = 0.009817477
+minTheta = 0.58905
+maxTheta = pi-minTheta
+numTheta = int(ceil((maxTheta-minTheta)/deltaTheta))
+if verbose:
+    print("minTheta = ", minTheta)
+    print("maxTheta = ", maxTheta)
+    print("numTheta = ", numTheta)
+    print("Nplanes = ", Nplanes)
+    print("rmin = ", rmin)
+    print("activeTotal = ", activeTotal)
+    print("readoutLayerRadialLengths = ", readoutLayerRadialLengths)
+    print("inclination (deg) = " , inclination_degree)
+    print("inclinedTotal = ", inclinedTotal)
+    print("passiveThickness = ", passiveThickness)
+
+#PCB dimensions [mm]
+hhv = 0.1
+hs = 0.17
+t = 0.035
+w = 0.127
+ws = 0.250
+#hm = 0.250
+hm = 0.2075
+pcbThickness = 7 * t + 2 * hhv + 2 * hs + 2 * hm #mm
+print("pcbThickness: %f"%pcbThickness)
+#constants:
+# distance from signal trace to shield (HS) - from impedance vs. trace width vs. distance to ground layer 2D plot (Z = 50 Ohm)
+# trace width (W) - min value
+# trace thickness (T) - min value
+# distance from shield to the edge of PCB
+#http://www.analog.com/media/en/training-seminars/design-handbooks/Basic-Linear-Design/Chapter12.pdf, page 40
+#signal trace
+epsilonR = 4.4 # PCB
+#conversion factor: 1 inch = 25.4 mm
+inch2mm = 25.4
+#capa per length from maxwel
+capa_per_mm = 0.123 # pF/mm
+capa_per_mm_stripLayer = 0.062 # pF/mm
+# multiplicative factor
+# factor two because we merge two phi cells together, another factor 2 because we have two 1) signal pad / shield capa  2) HV plate / absorber capa per cell
```

Comment:
It looks like there is indeed a small mistake here, I had changed this to a factor 2  instead of 4 because I moved to capacitance derived from ANSYS Maxwell which already takes into account the "factor 2 because we have two  signal pad / shield capa". But for the part "HV plate / absorber capa per cell" this factor 2 should be there. It won't make a big difference but it seems not correct per se. 

I propose to refactor this a bit: apply the factor 2 for the "HV plate / absorber capa per cell", then apply the factor "numberOfMergedCellsInPhi" which now depends on the layer to all capacitances. The trace capacitance is in principle already taken into account in the noise estimation.

That being said, this very detailed treatment is probably an overkill now that we now that the noise is not a big deal. Let's discuss it when possible.

Reply:
Do I understand correctly that the factors to apply are:
* shield: x2 (extra factor 2 already included from ANSYS)
* trace: x4
*  detector: x2 
?

Reply:
Hi Giovanni, not exactly. I am afraid we have to refactor this code a bit if we want to have a proper treatment for an arbitrary cell merging strategy. One way to do is to have lists of multiplicative factors `n_cells_merged_phi` and `n_cells_merged_theta` which should be applied (both) to the 'final cell capacitance'. To get the final cell capacitance, just sum the detector capacitance (`2 * epsilon0 * epsilonRLAr * area / distance`, explicitly adding here the missing factor 2 for the fact that we have two singal pad/absorber capa per cell) and the capacitance from the shields (the value for the "shield capacitance per mm" from Maxwell should be cross-checked and re-computed with the upcoming new PCB design but you can certainly start with what is there). The contribution from the "trace" (understand here the transmission line itself) is in principle already included in Martin's noise computation.

Reply:
changed in latest commit, please have a look. I have some doubt about the shield capacitance which is quite bigger than before due to the factor n_cells_merged_theta = 4 which was missing previously